packages/httpsnippet-client-api/test/__datasets__/operationid-with-underscores/output.js

```diff
@@ -0,0 +1,5 @@
+const sdk = require('api')('https://example.com/openapi.json');
+
+sdk.anythingOperation()
```

Comment:
Though the `operationId` was `anything_Operation` we're cleaning it up to `anythingOperation` in `oas` because it's not alphanumeric ([code here](https://github.com/readmeio/oas/blob/main/src/operation.ts#L339)). Maybe underscores in these should be left alone?